<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>presupuesto_ed</class>
 <widget class="QDialog" name="presupuesto_ed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edición de presupuesto</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="titulolabel">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PRESUPUESTO EJERCICIO XXXXX</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>SELECCIÓN DE CUENTAS</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Cuentas:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="listacuentaslineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="refrescarpushButton">
           <property name="toolTip">
            <string>Refrescar</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="salirpushButton">
           <property name="text">
            <string>&amp;Salir</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="adictodaspushButton">
           <property name="text">
            <string>Añadir todas</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="adicselpushButton">
           <property name="text">
            <string>&amp;Añadir selección</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout">
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="importarlabel">
           <property name="text">
            <string>Importar cuentas de presupuesto, ejercicio:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ejerciciosimportarcomboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="importarpushButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Refrescar</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="title">
            <string>Base del presupuesto</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <layout class="QGridLayout">
              <item row="0" column="0">
               <widget class="QRadioButton" name="baseejercicioradioButton">
                <property name="text">
                 <string>Importes base ejercicio</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="ejerciciobasecomboBox"/>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="basepresupuestoradioButton">
                <property name="text">
                 <string>Importes base presupuesto:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" colspan="2">
               <widget class="QComboBox" name="ejerciciopresupuestobasecomboBox"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QRadioButton" name="base0radioButton">
              <property name="text">
               <string>Presupuesto base cero</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Porcentaje incremento por defecto, cuentas añadidas</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignJustify|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="incrementolineEdit">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="imprimirpushButton">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>35</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Imprimir</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="keme.qrc">
             <normaloff>:/iconos/graf/printer.png</normaloff>:/iconos/graf/printer.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="pretableView">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="borratodopushButton">
           <property name="text">
            <string>Borrar todo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="borralinpushButton">
           <property name="text">
            <string>Borrar línea</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editarpushButton">
           <property name="text">
            <string>Editar importes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refrescaprepushButton">
           <property name="text">
            <string>Refrescar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections/>
</ui>
